app.go

```diff
@@ -18,6 +18,9 @@ import (
 	"flamingo.me/flamingo/v3/framework/flamingo"
 	"flamingo.me/flamingo/v3/framework/opencensus"
 	"flamingo.me/flamingo/v3/framework/web"
+
+	// Register automaxprocs to automaticly set GOMAXPROCS in container environments
+	_ "go.uber.org/automaxprocs"
```

Comment:
This is a bad place for registration, because it is not controllable for anyone using `flamingo.App`. Also, it is not necessary to use `flamingo.App`, so this might, or might not, work.

Instead, please create a small core module, maybe `core/runtime`, and include that as a default module in the `flamingo.App` modules.
The modules `Configure` can then use `sync.Once` to do `maxprocs.Set(maxprocs.Logger(logger.Printf))`, which sets the correct maxprocs settings and uses the Flamingo logger (eventually ensuring that there is proper JSON output).

Reply:
@thebod i integrated the changes. the failing GolangCi looks like a bug in GolangCi. in the analyse step is an depracated param used which results in an invalid json...